.github/workflows/conda.yml

```diff
@@ -58,6 +59,6 @@ jobs:
           ANACONDA_API_TOKEN: ${{ secrets.DASK_CONDA_TOKEN }}
         run: |
           # install anaconda for upload
-          mamba install anaconda-client
+          mamba install -c conda-forge anaconda-client
 
           anaconda upload --label dev noarch/*.tar.bz2
```

Comment:
Tangential, but wonder if there's a way to upload the package using `conda mambabuild` here instead, since that command would've seemingly uploaded the package in the prior step if `--no-anaconda-upload` wasn't set? This would save us the additional install of `anaconda-client` altogether.